unix/src/machine_stats/modules/proc_stats.py

```diff
@@ -92,9 +92,11 @@ def parse_status(process_path):
     status = dict()
     with open(process_path + "/status") as proc_status:
         for line in proc_status:
-            name, value = line.split(":")
-            name = name.lower().strip()
-            status[name] = value.strip()
+            result = line.split(":")
+            name = result[0].lower().strip()
+            value = result[1].strip()
```

Comment:
could consider simplifying and avoiding overhead  
```suggestion
            stats[name] = result[1].strip()
```

Reply:
I thought of it too. but reverted back to a verbose approach. 
Lets do it.